cmake/CVSLoggerConfig.cmake.in

```diff
@@ -0,0 +1,8 @@
+@PACKAGE_INIT@
+include(CMakeFindDependencyMacro)
+
+find_dependency(spdlog)
+find_dependency(PkgConfig)
+pkg_check_modules(systemd QUIET IMPORTED_TARGET libsystemd)
```

Comment:
why QUIET instead of REQUIRED? Older version with QUIET fails to compile exactly because of missing systemd on one clean system